.coveralls.yml

```diff
@@ -1 +1 @@
-repo_token: C2ZR5FGcdb7ExbsZcEWBnRvYS9IlKOslx
+repo_token: NODTwxCn99KpmU4XBJgrSWmzlcjR0i4lz
```

Comment:
I'm also seeing that the reporting is functioning for Coveralls: https://coveralls.io/github/pulibrary/pulfalight

Reply:
oh, hm. Should I delete/gitignore this file? I thought I had. coveralls recommends keeping the token secret. I'm not entirely sure how the token relates to the integration if it's not intended to be included in the repository.